file-validation.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * Validation applied to verify wether the bot should execute or skip the file specified
+ */
+
+/**
+ * @param $temp_file_name
+ * @param $file_name
+ *
+ * Validates if the file is valid to be scanned by vip-go-ci
+ *
+ * @return array
+ *	[
+ *		'issues' =>
+ *		[
+ *			'max-lines' => [$file_name],
+ *		],
+ *		'total' => 1
+ *	]
+ */
+function vipgoci_validate( $temp_file_name, $file_name ) {
+	$validation_result = array( 'total' => 0 );
+
+	if ( false === vipgoci_is_number_of_lines_valid( $temp_file_name, $file_name ) ) {
+		$validation_result[ 'issues' ][ VIPGOCI_VALIDATION_MAXIMUM_LINES ] = [ $file_name ];
+		$validation_result[ 'total' ] = count( $validation_result[ 'issues' ] );
+	}
+
+	return $validation_result;
+}
+
+/**
+ * @param $temp_file_name
+ * Validates if the number of lines is lower than the maximun
+ *
+ * @return bool
+ */
+function vipgoci_is_number_of_lines_valid( $temp_file_name, $file_name ) {
+	/**
+	 * Calculates the file number of lines
+	 *
+	 * if it is > than the default limit (defined at defined)
+	 *
+	 * the bot won't scan it
+	 */
+	$cmd = sprintf(
+		'wc -l %s | awk \'{print $1;}\'',
+		escapeshellcmd( $temp_file_name )
+	);
+
+	$output = ( int ) vipgoci_runtime_measure_shell_exec(
```

Comment:
Maybe check if `$output` is a numeric before casting, and if it is not print error? Otherwise error messages might lead to strange problems later on as casting a string can result in a non-zero integer value.

Reply:
I made a small modification to avoid that casting. Can you take a look and let me know what your thoughts are?

Reply:
Looks good! :)